bin/templates/cordova/lib/build.js

```diff
@@ -422,11 +477,15 @@ function parseOpts(options, resolvedTarget) {
 module.exports.runClean = function(options) {
     var opts = parseOpts(options);
     var builder = builders[opts.buildMethod];
-    return builder.prepEnv()
+    return builder.prepEnv(opts)
     .then(function() {
-        return builder.clean(opts.extraArgs);
+        return builder.clean(opts);
     }).then(function() {
         shell.rm('-rf', path.join(ROOT, 'out'));
+    }).then(function() {
+        ['debug', 'release'].forEach(function(config) {
+            removeIfExists(path.join(ROOT, config + SIGNING_PROPERTIES));
```

Comment:
hmm, maybe for this reason you should pass the parameters on the command line as gradle properties so that there is no file to clean up. Note that it's unlikely the ANT workflow is important anymore, as gradle is the new default and seems to work much better.